src/lib/completion_test.rs

```diff
@@ -0,0 +1,90 @@
+use std::fs;
+use std::path::Path;
+use std::io::Cursor;
+
+#[cfg(test)]
+mod tests {
```

Comment:
i don't think this block is needed as this file is included only for cfg test to being with

Reply:
blocked has been removed